prov/rxm/src/rxm_cq.c

```diff
@@ -414,7 +414,8 @@ static int rxm_lmt_send_ack(struct rxm_rx_buf *rx_buf)
 		return -FI_EAGAIN;
 	}
 
-	if (!(tx_entry = rxm_tx_entry_get(&rx_buf->ep->send_queue))) {
```

Comment:
Was this a warning in windows? I'm fine with the change. Just curious.

Reply:
unfortunately, the Windows compiler complains that we are doing an assignment in the condition.
I tried to disable this warning via `#pragma`. But in this case the code like this
```
if (a = 5) { ... }
```
looks good for compiler w/o any warnings.
I guess it's more better to restrict some minor convention for the C syntax than allow a potentially incorrect condition w/o any warnings

Reply:
Just as an example for the AppVeyor CI testing:

```
c:\projects\libfabric\prov\util\src\util_poll.c(102): warning C4706: assignment within conditional expression
```

Reply:
Thanks for the clarification!